lib/resources/datav2/rest_v2.js

```diff
@@ -65,6 +64,30 @@ async function* getDataV2(endpoint, symbol, options = {}, config) {
   }
 }
 
+async function* getMultiDataV2(endpoint, options = {}, config) {
+  let pageToken = null;
+  while (true) {
+    let params = options;
+    Object.assign(params, { limit: options.limit, page_token: pageToken });
+
+    const resp = await dataV2HttpRequest(
+      `/v2/stocks/${endpoint}`,
+      params,
+      config
+    );
+    const items = resp.data[endpoint];
+    for (const symbol in items) {
+      for (const data of items[symbol]) {
+        yield { symbol: symbol, data: data };
```

Comment:
why return an iterator here instead of an array? don't iterators lack reduce, filter, and similar methods that are often used?